bakerydemo/templates/blocks/blockquote.html

```diff
@@ -1,7 +1,10 @@
 {% load wagtailimages_tags %}
 
-
-<blockquote><p class="text">{{ self.text }}</p>
-<footer><p class="attribute_name">{{ self.attribute_name}}</p></footer>
+<blockquote>
+    <p class="text">
+        {{ self.text }}<br />
+        {% if self.attribute_name %}
```

Comment:
Hmm, I thought it was unusual to use `footer` in a blockquote, but you should go ahead and do whatever you like on this one.